internal/website/snippets/tutorials/deploy/guestbook.md

```diff
@@ -0,0 +1,5 @@
+Guestbook is a sample application that records visitors' messages, displays a
+cloud banner, and an administrative message. The main business logic is written
```

Comment:
Nit: The first sentence feels a bit off to me, since it has sublist in a list, both in a single level with commas. This is not about the oxford comma :)

Something like:

"Guestbook is a sample application that records visitors messages and displays a cloud banner with an administrative message" would be clearer, IMHO

Reply:
Nice! Done.